index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <header id="header">
+    <img id="header-img"
+      src="https://www.google.com.mx/url?sa=i&url=https%3A%2F%2Fwww.gearbots.org%2Fengineering-challenge%2F&psig=AOvVaw0sH6LZ5-gCxCw7f8rawnff&ust=1682532750762000&source=images&cd=vfe&ved=0CBEQjRxqFwoTCOiA6YrRxf4CFQAAAAAdAAAAABAI"
+      alt="">
+    <nav id="nav-bar">
+      <a href="#header" class="nav-link"></a>
+      <a href="#header" class="nav-link"></a>
+      <a href="#header" class="nav-link"></a>
+    </nav>
+  </header>
+
+  <video id="video" src="video.com">
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <input type="email" id="email" placeholder="enter you email" name="email">
+      <input type="submit" id="submit">
+    </form>

```

Comment:
Please add video and form in a section tag to make it more sematic.
<section> //video and form contents</section>

Reply:
Good job! Please add video and form in a section tag to make it more sematic. The section tag is a better way of separating your code. Having a more semantic code makes its content more distinguishable from each other. Hence other people working on the project can get better understanding of the code.
`<section><!-- video and form contents --></section>`